README.md

```diff
@@ -1,20 +1,39 @@
 # Mapbox Annotation Extension for iOS
 
-The Mapbox Annotation Extension is lightweight library you can use with the Mapbox Maps SDK to quickly add basic shapes to a map.
+The Mapbox Annotation Extension is a lightweight library you can use with the Mapbox Maps SDK for iOS to quickly add basic shapes to a map.
```

Comment:
"basic shapes, icons, and other annotations" for greater clarity. I know iOS has a specific idea of annotations but since we're already calling this the Annotation Extension, I think we should run with it and make "annotations" encompass everything that can be added thru the extension. see the blog paper doc comment history where @captainbarbosa and I discuss this